### DISCUSSION TITLE
Diskussion der Ergebnisse

### DISCUSSION THREAD

**Discussion by Stammkundschaft** [yannick0404/GrupppenarbeitWA#24] [General]
Ich fange mal an:

Alter schwankt im Datensatz so um die 25 Jahre rum, wobei es nicht wirklich große Ausreißer gibt (Range von 20 bis 30 Jahren).
Leute, die im Mathe-LK waren, sind im Schnitt aelter als die, die nicht drin waren.
Aus dem Korrelationsplot kann man einen leicht positiven Zusammenhang zwischen dem Alter und dem Studienfach Info herauslesen.

**Comment by edos1** [comment]
Hier mal mein Beitrag dazu:
Zunächst zum Alter: Dabei stimmt der Mittelwert ja ziemlich genau mit dem aus der Simulation überein. Die Zusammenhänge des Alters mit anderen Variablen müssten, meiner Einschätzung nach, zufällig sein, denn wir haben in der Simulation keine solchen einprogrammiert.
Bei der Auswertung der Beziehung zwischen Interesse für Info und Interesse für Mathe schreibt ihr, dass die Korrelation von ca. -0.13 auf einen geringen Zusammenhang hindeutet. Da wäre ich etwas skeptischer -0.13, ist ja schließlich nicht wirklich viel. In der Simulation haben wir da auch keine direkten Abhängigkeiten eingebaut, allenfalls über die Studienfächer sind da indirekte Zusammenhänge denkbar. Die Interessen haben wir nämlich von den Fächern abhängig gemacht.
Auch bei der Korrelation zwischen Interesse an Info und dem Alter wäre ich vorsichtig.
Die Korrelation zwischen Interesse an Mathe und der binären Variable Mathe LK hingegen scheint mir ganz logisch. Die Variable hängt in der Simulation nämlich unmittelbar von dem Interesse an Mathe ab (Das Interesse an Mathe geteilt durch 10 ist die W'keit Mathe LK belegt zu haben).

Das waren erstmal die Aspekten, die mir so auf die Schnelle aufgefallen sind. Was ist euch noch aufgefallen oder seht ihr irgendwas von den Sachen, die ich geschrieben habe anders?

**Reply by TimonM1705** [reply]
Mit der leicht negativen Korrelation von -0.13 hast du natürlich recht. Das ist ein so geringer Wert, der führt eigentlich zu keiner Aussage. Das wollte ich im Skript 4 einfach erwähnen, dass dieser Korrelationswert herausgegeben wird.

**Comment by TimonM1705** [comment]
Mir wäre jetzt noch aufgefallen, dass ein eher überdurchschnittliches Interesse an dem Fach Mathe besteht. Der Median liegt bei 5.0 und das arithmetische Mittel bei 4.65., sowie wie Quantile bei 4 und 6, was auf einer Skala von 1-7 recht hoch ist. Interesse an Informatik ist eher durchschnittlich bis überdurchschnittlich verteilt. Die Werte bezogen auf die Fächer Mathe und Informatik machen auch Sinn, da es hier schließlich um die Studiengänge "Data Science", "Statistik", "Mathematik" und Informatik geht. In allen Studiengängen geht es viel um Mathe, deswegen sollte auf jeden Fall Interesse an Mathe bestehen. Allerdings geht es in den meisten Studiengängen eher ein bisschen weniger um Informatik und deswegen ist auch das Interesse daran geringer. Außerdem ist das Fach "Informatik", das am wenigsten belegte Fach, sodass in Relation, das Interesse an Informatik geringer sein sollte.

**Reply by edos1** [reply]
Ja das mit dem Informatik Interesse stimmt schon. Es wäre wirklich realistischer, wenn das ein wenig niedriger wäre.

**Reply by yannick0404** [reply]
Ja das überdurchschnittliche Interesse an Mathematik ist ebenfalls gewünscht, da es sich, wie du bereits erwähntest, um eher mathematische Studiengänge
handelt.

**Comment by TimonM1705** [comment]
Bei der Beziehung zwischen Interesse für Mathe und Interesse für Info besteht nur eine sehr geringe negative Korrelation, das stimmt. Allerdings liegt die Kovarianz bei -0.44. Dies schließt darauf, dass, wenn das Interesse an dem einen Fach zunimmt, das Interesse an dem anderen Fach abnimmt. Dies könnte auch stimmen, da Personen, die zum Beispiel das Fach " Informatik" wählen, lieber programmieren und eher weniger mit Mathe zutun haben wollen. Dies hat man ja auch bereits mal gehört, dass die Mathemodule dort eher gezwungenermaßen gemacht werden. Und genauso verhält sich das bestimmt bei den Mathestudiengängen. Dort will man wahrscheinlich eher weniger Informatik machen.

**Reply by edos1** [reply]
Die Korrelation ist doch einfach nur die Kovarinz normiert, so dass sie immer zwischen -1 und 1 liegt, oder vertue ich mich da? Deshalb würde ich sagen, dass man bei dem Zusammenhang auf die Korrelation schauen sollte und die ist ja nicht aussagekräftig. Also wäre ich da, wie gesagt, vorsichtiger bei der Interpretation.

**Reply by TimonM1705** [reply]
Das stimmt schon. Die Kovarianz gibt dir die Richtung der linearen Beziehung der zwei Variablen an und die Korrelation die Stärke, sowie auch die Richtung, der Beziehung der Variablen. Deswegen hatte ich jetzt mal meine Hypothese aufgestellt. Aber verbessert mich da ruhig, wenn ich falsch liege.

**Reply by yannick0404** [reply]
Ja dahast du recht, nur ist , wie @edos1  bereits sagte, die Korrelation aussagekräftiger. Ich denke aber auch, dass man, wenn auch nicht unbedingt von einer starken Korrelation, aber durchaus von einer Kausalität sprechen kann. Die von dir angesprochene Korrelation wurde jedenfalls nicht bewusst von uns simuliert (oder hast du daran gedacht @edos1 ?).

**Reply by edos1** [reply]
Nein das haben wir nicht bewusst eingebaut

**Comment by TobiasPytlik** [comment]
Wie bereits in den anderen Kommentaren schon erwähnt lief eure Datensatzerstellung sehr gut, dass heißt die Aufgabenstellung wurde gut umgesetzt und wir konnten zum Beispiel die Korrelation zwischen dem Interesse an Mathe und dem belegen des Mathe LKs feststellen. Trotzdem würde ich gerne anmerken, dass bei Betrachtung des Balkendiagramms zum Studiengang oder einfach ein Einblick in table(daten$Studiengang) deutlich zeigt, dass Informatik mit Abstand am wenigsten belegt wurde. Statistik und Data Science liegen wie gewünscht sehr nah beisammen und machen den größten Anteil aus, danach sollte jedoch Informatik folgen und Mathe mit der geringsten Wahrscheinlichkeit auftreten. Liegt hier einfach eine Verwechslung beim Erstellen des Datensatzes vor oder haben wir hier nur ein Ergebnis, welches mit sehr geringer Wahrscheinlichkeit auftritt?

**Comment by edos1** [comment]
Ja, da liegt wohl eine kleine Verwechselung vor. Wir haben in der Simulation dem Fach Mathe eine W'keit von 4/17 und Informatik eine W'keit von 1/17 zugeordnet. Es sollte natürlich eigentlich umgekehrt sein. Damit liegt hier also kein Ausnahmefall vor, sondern es gab nur eine kleine Verwechselung.

**Reply by TobiasPytlik** [reply]
Alles klar, dann sind wir aber doch jetzt eigentlich fertig oder? Wir haben uns ja über alle Aspekte ausgetauscht und konnten nachweisen, welche Zusammenhänge ihr in den Datensatz eingebaut habt

**Reply by yannick0404** [reply]
Ja, hat ja insgesamt gut geklappt : )

**Reply by edos1** [reply]
Jo, würde ich auch sagen. Danke für die gute Zusammenarbeit!
